spec/integration/_dev/benchmark/system.scenario.spec.yml

```
@@ -65,16 +65,11 @@ spec:
           type: object
           additionalProperties: false
           required:
-            - size
+            - tot_events
```

Could this be called `total_events`? Or it is too late to rename?
```suggestion
            - total_events
```

---

`tot_events` is already part of the spec, but it's not used anywhere yet but for this PR and one in `elastic-package`.
if we include in the 3.0.0 we can rename taking the opportunity of the breaking change, up to you

---

If this is not being used, let's rename to `total_events` if possible, as we are doing this a breaking change in any case.

---

done!